security_groups.tf

```diff
@@ -18,21 +19,47 @@ resource "aws_security_group" "eq_alb" {
   }
 
   tags {
-    Name        = "${var.env}-eq-alb-access"
+    Name        = "${var.env}-eq-alb"
```

Comment:
I think we should set the `Name` tag to `{env}-eq-ecs`, as:
- The security group name captures the specific name
- We'll be able to easily identify all the security groups for the ECS cluster